tools/pkglint/lib/rules.ts

```diff
@@ -869,6 +884,7 @@ function cdkModuleName(name: string) {
       distName: `aws-cdk.${pythonName}`,
       module: `aws_cdk.${pythonName.replace(/-/g, '_')}`,
     },
+    goRootModuleName: 'github.com/aws/aws-cdk-go',
```

Comment:
Possible alternative (take it or leave it):

```suggestion
    goDistributionRepository: 'github.com/aws/aws-cdk-go',
```

Reply:
Changing to `goRepositoryName` to align with the proposal in https://github.com/aws/jsii/issues/2632